test_runner/performance/test_pageserver_pagebench.py

```diff
@@ -0,0 +1,111 @@
+import json
+from pathlib import Path
+
+import fixtures.pageserver.many_tenants as many_tenants
+import pytest
+from fixtures.benchmark_fixture import NeonBenchmarker
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import (
+    NeonEnv,
+    NeonEnvBuilder,
+    PgBin,
+    last_flush_lsn_upload,
+)
+from fixtures.pageserver.utils import wait_until_all_tenants_state
+
+
+def many_small_tenants_snapshot(
+    neon_env_builder: NeonEnvBuilder,
+    n_tenants: int,
+) -> NeonEnv:
+    def setup_template(env: NeonEnv):
+        # create our template tenant
+        config = {
+            "gc_period": "0s",
+            "checkpoint_timeout": "10 years",
+            "compaction_period": "20 s",
+            "compaction_threshold": 10,
+            "compaction_target_size": 134217728,
+            "checkpoint_distance": 268435456,
+            "image_creation_threshold": 3,
+        }
+        template_tenant, template_timeline = env.neon_cli.create_tenant(set_default=True)
+        env.pageserver.tenant_detach(template_tenant)
+        env.pageserver.tenant_attach(template_tenant, config)
+        # with env.endpoints.create_start("main", tenant_id=template_tenant) as ep:
+        #     pg_bin.run_capture(["pgbench", "-i", "-s5", ep.connstr()])
+        #     last_flush_lsn_upload(env, ep, template_tenant, template_timeline)
+        ep = env.endpoints.create_start("main", tenant_id=template_tenant)
+        ep.safe_psql("create table foo(b text)")
+        for _ in range(0, 8):
+            ep.safe_psql("insert into foo(b) values ('some text')")
+            last_flush_lsn_upload(env, ep, template_tenant, template_timeline)
+        ep.stop_and_destroy()
```

Comment:
Did you move away from the commented out pgbench scale 5 because it is much larger than the 8 rows or did it have some adverse other effects for the test...?

These could be of course tuned later.